frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -298,9 +304,19 @@ const PropertyListView: React.FC = () => {
           columns={columns}
           data={data || []}
           loading={data === undefined}
+          sort={sorting}
           pageIndex={pageIndex}
           onRequestData={handleRequestData}
           pageCount={pageCount}
+          onSortChange={(column: string, direction: SortDirection) => {
+            if (!!direction) {
+              setSorting({ ...sorting, [column]: direction });
+            } else {
+              const data: any = { ...sorting };
+              delete data[column];
+              setSorting(data);
```

Comment:
why do you create it and then immediately delete it?

Reply:
Just removing the sorting key from the sort object because the sort direction is undefined